packages/expo-location/android/src/main/java/expo/modules/location/taskConsumers/LocationTaskConsumer.java

```diff
@@ -159,6 +159,10 @@ public void onFinished(Map<String, Object> response) {
     return true;
   }
 
+  public static boolean useForegroundService(Map<String, Object> options) {
```

Comment:
I'm not sure if this is common. I did this to keep the `FOREGROUND_SERVICE_KEY` "in sync" with validation from the module file. We can also:
1. Copy the string/constant to the module
2. Expose this string/constant to the module

Reply:
Actually, I like this ;)